package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "Erase.bg",
-	"version": "2.0.1",
+	"version": "18",
```

Comment:
why this is `18` should be minor version bump right?

Reply:
Cannot have minor versions as figma doesnt allow to publish minor version